app/assets/stylesheets/boards.scss

```diff
@@ -0,0 +1,27 @@
+ 	.board {
+ 		width: 60%;
+    	height: 70%;
+    	border: 3px solid #333;
+    	margin: 10px auto 10px auto;
+
```

Comment:
In the ticket it said to not use bootstrap, I thought about it myself. I think now that I built it without bootstrap maybe we can check with Dave and see if I can use now ?